x-pack/plugins/actions/server/plugin.ts

```diff
@@ -163,17 +163,17 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
     plugins: ActionsPluginsSetup
   ): Promise<PluginSetupContract> {
     this.licenseState = new LicenseState(plugins.licensing.license$);
-    this.isESOUsingEphemeralEncryptionKey =
-      plugins.encryptedSavedObjects.usingEphemeralEncryptionKey;
+    this.isESOAvailable = !!plugins.encryptedSavedObjects;
 
-    if (this.isESOUsingEphemeralEncryptionKey) {
+    if (!this.isESOAvailable) {
       this.logger.warn(
-        'APIs are disabled because the Encrypted Saved Objects plugin uses an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or use the bin/kibana-encryption-keys command.'
+        'APIs are disabled because the Encrypted Saved Objects plugin is not available. Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or use the bin/kibana-encryption-keys command.'
       );
+    } else {
+      setupSavedObjects(core.savedObjects, plugins.encryptedSavedObjects!);
```

Comment:
This looks wrong - if ESO is disabled presumably we still need to setup the actions saved objects?

Reply:
It's not possible because we will get a Kibana start error that we are not able to register alert saved object type, because it is hidden.